package.json

```diff
@@ -39,9 +39,11 @@
     "aguid": "^1.0.4",
     "bignumber.js": "^2.4.0",
     "canonical-json": "0.0.4",
+    "co-request": "^1.0.0",
```

Comment:
I think we should use `superagent` instead of `co-request` because as far as I know it's a lighter dependency and we don't need all the extra functionality request includes